app/src/main/java/org/grapheneos/pdfviewer/PdfViewer.java

```diff
@@ -218,7 +218,7 @@ public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceReque
                     headers.put("Content-Security-Policy", CONTENT_SECURITY_POLICY);
                     headers.put("Permissions-Policy", PERMISSIONS_POLICY);
                     headers.put("X-Content-Type-Options", "nosniff");
-                    response.setResponseHeaders(headers);
+                    Objects.requireNonNull(response).setResponseHeaders(headers);
```

Comment:
I would rather not use this approach right now.

Reply:
Removed Object.requireNonNull